css-canvas-medical/styles.css

```diff
@@ -0,0 +1,276 @@
+.container {
+  max-width: 1080px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+}
+
+.col-one-half {
+  flex-basis: calc(100% / 2);
+}
+
+.col-one-third {
+  flex-basis: 360px;
```

Comment:
this needs to be a percentage based value, should be calc(100%/3)

Reply:
![canvas-medical](https://user-images.githubusercontent.com/90743100/171305625-582799e7-5b6f-4e40-b3d6-9c04f29216ca.png)